net/libfab/libfab.h

```diff
@@ -0,0 +1,55 @@
+/* -*- C -*- */
+/*
+ * Copyright (c) 2020 Seagate Technology LLC and/or its Affiliates
```

Comment:
This work is started in 2020, but the first commit will be 2021.
Shall we modify it to 2021?
Or 2020-2021?

Reply:
Modified to 2021